index.html

```diff
@@ -155,7 +155,7 @@
         Geography night
       </h1>
 
-      <h2 data-i18n="event-date">13th - 14th of April 2023</h2>
+      <h2 data-i18n="event-date">April 13-14, 2023</h2>
```

Comment:
```suggestion
      <h2 data-i18n="event-date">April 14, 2023</h2>
```

We used to make this two days but the official day is only the 14th https://www.geonight.net/